### PR TITLE
Move "Why this is important" to before requirements

### PR DESCRIPTION
**Pull request by ericherman**
fixes: #560

### CODE REVIEW

**Comment by Ainali** [comment]
Very nice and good to get started on this! I was looking at the videos we made for our course earlier and inspired by that I combined it with your suggestion for Bundle policy and code to this:

> When someone wants to collaborate in, or reuse, a codebase they need contextual information about the purpose and objective the codebase to be able to evaluate whether to adopt that codebase in their context. To get that level of understanding it is necessary that they have access to both the source code and the policy documents. That access is fundamental to understanding what problems a codebase is trying to solve and how it sets out to solve them. It is also required in order to be able to understand what process changes an organization must choose to make or how to contribute additional configurability to the existing solution in order to adapt it to the new context.
> 
> Therefore, providing access to both source code and policy documentation provides building blocks for others to reimplement the codebase in their local context or as a starting point to contribute to the development of the codebase.

I am not sure that this is a clearly different style, or if it is just a bit reordered and longer text, but at least to me, it reads a bit smoother.

**Comment by Ainali** [comment]
To recap some discussion that we have had this morning, these new lead section should be less of a direct answer to the question "Why is this important" but rather be something of a rationale for the entire criterion (almost like the recitals in EU directives or pre-ambles in other legislative documents).
Therefore, we should check that all requirements are possible to derive from the lead section. If not, it might suggest that we need to add more to the pre-amble, or that the requirement really fits better in another criterion. (It's possible, but less likely, that this will also reveal some requirements that are really beyond the scope of the standard.)

**Comment by ericherman** [comment]
This branch now contains a rough draft of what it might look like.

Many of the sentences need work, but we hope the ideas are there.

We have identified several issues which suggest that some requirements might fit better under different criteria:
* #814
* #815
* #817
* #819
* #820

**Review by Ainali — APPROVED** [review]

**Comment by ericherman** [comment]
https://blog.publiccode.net/community%20call/2023/02/02/notes-from-community-call-3-february-2023.html

File: readers-guide.md [hunk 1]
```diff
@@ -11,6 +11,10 @@ It is common for individuals to have duties which span multiple roles.
 
 Below is a brief explanation of each of these sections and how they are used within the criteria of the Standard.
 
+## Preamble
```
Comment (ElenaFdR) [inline readers-guide.md:14]:
```suggestion
## Introduction
```

'Preamble' is a less common word than 'introduction', and carries more (unnecessary) overtones of legal jargon.

**Comment by Ainali** [comment]
I noticed that in many places it is no longer the first time a term is used that ew are linking to the glossary. I think we need to look through each criterion and see if the glossary links are in the right place.

**Comment by ericherman** [comment]
Ah yes, quite right. Let's do a pass at that.

**Review by Ainali — APPROVED** [review]

**Pull request merged**